proto/poktroll/supplier/tx.proto

```diff
@@ -45,17 +45,30 @@ message MsgUpdateParams {
 message MsgUpdateParamsResponse {}
 
 message MsgStakeSupplier {
-  option (cosmos.msg.v1.signer) = "address"; // https://docs.cosmos.network/main/build/building-modules/messages-and-queries
-  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the supplier using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding
-  cosmos.base.v1beta1.Coin stake = 2; // The total amount of uPOKT the supplier has staked. Must be ≥ to the current amount that the supplier has staked (if any)
-  repeated poktroll.shared.SupplierServiceConfig services = 3; // The list of services this supplier is staked to provide service for
+  option (cosmos.msg.v1.signer) = "sender"; // https://docs.cosmos.network/main/build/building-modules/messages-and-queries
+
+  string sender = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the message sender (i.e. owner or operator) using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding
```

Comment:
I think we can omit `using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding`.

Too much. DItto everywhere.